projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-history/replenishment-order-history.component.html

```diff
@@ -0,0 +1,222 @@
+              {{ 'orderHistory.startOn' | cxTranslate }}
+            </th>
+            <th scope="col">
+              {{ 'orderHistory.frequency' | cxTranslate }}
+            </th>
+            <th scope="col">
+              {{ 'orderHistory.nextOrderDate' | cxTranslate }}
+            </th>
+            <th scope="col">
+              {{ 'orderHistory.total' | cxTranslate }}
+            </th>
+            <th scope="col"></th>
+          </thead>
+          <tbody>
+            <tr *ngFor="let order of replenishmentOrders.replenishmentOrders">
+              <td
+                class="cx-replenishment-order-history-code"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.replenishmentOrderId' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.replenishmentOrderCode }}</a
+                >
+              </td>
+              <td
+                class="cx-replenishment-order-history-placed"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.purchaseOrderNumber' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{
+                    order?.purchaseOrderNumber?.length > 0
+                      ? order?.purchaseOrderNumber
+                      : ('orderHistory.emptyPurchaseOrderId' | cxTranslate)
+                  }}
+                </a>
+              </td>
+              <td
+                class="cx-replenishment-order-history-status"
+                (click)="goToOrderDetail(order)"
+              >
+                <div class="d-md-none cx-replenishment-order-history-label">
+                  {{ 'orderHistory.startOn' | cxTranslate }}
+                </div>
+                <a
+                  [routerLink]="
+                    {
+                      cxRoute: 'replenishmentDetails',
+                      params: order
+                    } | cxUrl
+                  "
+                  class="cx-replenishment-order-history-value"
+                >
+                  {{ order?.firstDate | cxDate }}</a
+                >
+              </td>
+              <td
+                class="cx-replenishment-order-history-total"
```

Comment:
pretty weird the other `td` (3 of them) has the class `cx-replenishment-order-history-total`

I would say name it appropriately and add them in the stylesheet by stacking them or make the name generic as it is a class to be re-used.

BUT I took at the class, and it says to `text-align: right` however that's not what we want and it does get override to left from another styling